lib/facility.rb

```diff
@@ -1,14 +1,73 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :address,
+              :name,
+              :phone,
+              :services,
+              :collected_fees, 
+              :registered_vehicles,
+              :operating_hours,
+              :holiday_closures
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(office)
+    @address = office[:address]
+    @name = office[:name]
+    @phone = office[:phone]
+    @operating_hours = office[:operating_hours]
+    @holiday_closures = office[:holiday_closures]
     @services = []
+    @collected_fees = 0
+    @registered_vehicles = []
   end
 
-  def add_services(service)
+  def add_service(service)
     @services << service
   end
+
+  def register_vehicle(vehicle)
+    return p "Facility does not currently offer this service." unless @services.include?("Vehicle Registration")
```

Comment:
This is what we call a guard clause. Love it. I recommend putting a space after a guard clause so it's very clear, then the rest of the code below it should be at the same indentation as the guard clause as that's a one-liner and is not creating a block.